### PR TITLE
fix: database popup position fix while resizing window

### PR DESCRIPTION
**Pull request by gauravruhela07**

### CODE REVIEW

**Comment by gauravruhela07** [comment]
Hi @doodlewind, as per the PR the pop-up opens in the center(40% from the top)  of the screen and it resizes well while resizing screen. 

However, the position of the pop-up need to be at the point where the user clicks. I have fixed this as well in my latest commit. Please review and let me know if you have any more suggestion.

File: packages/blocks/src/_common/components/menu/menu.ts [hunk 1]
```diff
@@ -726,6 +726,23 @@ export const createPopup = (
       options?.onClose?.();
     }
   };
+
+  // logic to update pop-up position during resizing of screen
+  let currentScreenWidth = window.innerWidth;
+  window.addEventListener('resize', function () {
```
Comment (zzj3720) [inline packages/blocks/src/_common/components/menu/menu.ts:732]: Unsubscribe logic needs to be handled

Comment (zzj3720) [reply packages/blocks/src/_common/components/menu/menu.ts:732]: It is recommended to use autoUpdate in @floating-ui/dom

**Comment by doodlewind** [comment]
Have to close this in favor of #5457, thanks for your contribution anyway!

**Pull request closed without merging**